<ui version="4.0" >
 <class>MultiViewFrameMenu</class>
 <widget class="QWidget" name="MultiViewFrameMenu" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>18</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>18</width>
    <height>18</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>18</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="WindowCaption" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="ActiveButton" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text" >
      <string>A</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoRaise" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="SplitHorizontalButton" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text" >
      <string>Split Horizontal</string>
     </property>
     <property name="icon" >
      <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqSplitViewH12.png</iconset>
     </property>
     <property name="autoRaise" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="SplitVerticalButton" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text" >
      <string>Split Vertical</string>
     </property>
     <property name="icon" >
      <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqSplitViewV12.png</iconset>
     </property>
     <property name="autoRaise" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="MaximizeButton" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text" >
      <string>Maximize</string>
     </property>
     <property name="autoRaise" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="RestoreButton" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text" >
      <string>Restore</string>
     </property>
     <property name="autoRaise" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="CloseButton" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text" >
      <string>Close</string>
     </property>
     <property name="autoRaise" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pqComponents.qrc" />
 </resources>
 <connections/>
</ui>
